pkg/kubelet/cm/fake_container_manager.go

```diff
@@ -237,11 +236,3 @@ func (cm *FakeContainerManager) GetNodeAllocatableAbsolute() v1.ResourceList {
 	defer cm.Unlock()
 	return nil
 }
-
-func (cm *FakeContainerManager) PrepareResources(pod *v1.Pod, container *v1.Container) (*dra.ContainerInfo, error) {
-	return nil, nil
-}
-
-func (cm *FakeContainerManager) UnprepareResources(*v1.Pod) error {
-	return nil
-}
```

Comment:
How is windows handled with this functionality moved to the kubelet directly?

Reply:
That's a good question. May be moving these APIs to the Kubelet was not a good decision. Initially I left them in the container manager, which was probably better. I need to think about it.